src/compiler/utilities.ts

```diff
@@ -10295,7 +10295,7 @@ function skipTypeCheckingWorker(
     // If skipLibCheck is enabled, skip reporting errors if file is a declaration file.
     // If skipDefaultLibCheck is enabled, skip reporting errors if file contains a
     // '/// <reference no-default-lib="true"/>' directive.
-    return (options.skipLibCheck && sourceFile.isDeclarationFile ||

```

Comment:
Should this have been "true" to check if it was always on? Or were you looking to try and see if we never did this if things changed?

Reply:
Was trying to manifest opposite day